docs/content/preview/yugabyte-voyager/migrate/assess-migration.md

```diff
@@ -115,7 +183,7 @@ For the most accurate migration assessment, the source database must be actively
 
 1. Create a target YugabyteDB cluster as follows:
 
-    1. Create a cluster based on the sizing recommendations in the assessment report.
+    1. Create a cluster in [Enhanced Postgres Compatibility Mode](/preview/releases/ybdb-releases/v2024.1/#highlights) based on the sizing recommendations in the assessment report.
```

Comment:
This points to yb-db where the yugabyted instructions are present. (i.e. `--enable_pg_parity_early_access`). 
For YBA, the instructions are different - https://deploy-preview-23116--infallible-bardeen-164bc9.netlify.app/preview/releases/yba-releases/v2024.1/#highlights (you have to set GLAGs on the cluster). Can we link to both here ?

I'm not sure what to do about YBM/Aeon?

Reply:
I don't think we have instructions for YBM/Aeon yet. Ill link to it as soon as we have it. But will link for YBA and YBDB